lib/routes/account.js

```diff
@@ -1914,8 +1995,162 @@ module.exports = function (
             function () {
               reply({})
             },
+            function (err) {
+              reply(err)
+            }
+          )
+      }
+    },
+    {
+      method: 'GET',
+      path: '/recovery_email/emails',
+      config: {
+        auth: {
+          strategy: 'sessionTokenWithVerificationStatus'
+        },
+        response: {
+          schema: isA.array().items(
+            isA.object({
+              isVerified: isA.boolean().required(),
+              isPrimary: isA.boolean().required(),
+              email: validators.email().required()
+            }))
+        }
+      },
+      handler: function (request, reply) {
+        const sessionToken = request.auth.credentials
+        const uidHex = sessionToken.uid.toString('hex')
+        const userEmail = sessionToken.email
+        var emails
+
+        log.begin('Account.RecoveryEmailEmail', request)
+
+        customs.check(request, userEmail, 'accountEmails')
```

Comment:
Since this request is authenticated with a sessionToken, do we need to customs-check it?  It's not obvious what we want to guard against with this check.